django/core/management/commands/inspectdb.py

```diff
@@ -75,6 +76,7 @@ def handle_inspection(self, options):
                 "field names."
             )
             yield "from %s import models" % self.db_module
+
```

Comment:
@csirmazbendeguz these are minor clean up changes that will happen on the PR either by you or by a merger
- we want to limit the changed lines to keep the diff really tight to what's important, occasionally you have added a new line or removed a line, I will probably revert some of this (you can too if you like). This is considered formatting changes
- you have added `@tag("serial")` for your tests, this is not how we organise tests generally and needs to be removed
- the commits need to be squashed. This will likely be 1 commit (or if we can think of a nice way of splitting them we will)

Reply:
I added this for informational purposes 👍 

I forgot to say that I have tested this with a local project and tried out a number of things to try and see if I can find any issues (had a look in the admin,  reversed migrations, used them in `GeneratedFields`) and [this comment](https://github.com/django/django/pull/18123#discussion_r1616847144) is the only thing I'm concerned about. It looks really good ⭐